src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -95,7 +95,7 @@ public class CommandTestUtil {
     public static void assertCommandSuccess(Command command, Model actualModel,
                                             CommandResult expectedCommandResult, Model expectedModel) {
         try {
-            CommandResult result = command.execute(actualModel, new EventManager());
+            CommandResult result = command.execute(actualModel, actualModel.getEventManager());
```

Comment:
I changed this because it was causing my test cases to fail as it wasn't using the `EventManager` that i put in ;-;

Reply:
T_T

Reply:
Yea not sure why a empty event manager was used, I think some of the current pull requests also changed it to use actualModel.getEventManager()